docs/changelog.md

```diff
@@ -5,6 +5,8 @@
 * A new config flag `launchAdditionalService` has been added. True by default, it controls the launch of services
 additional service like Forkmon, Prometheus and Grafana, as well as the transaction spammer. Is set to false, only the
 Ethereum network (EL _and_ CL nodes) are launched
+* The testnet-verifier has been removed, and with it its associated config params: `waitForFinalization`, 
```

Comment:
ChangeLog entries should begin with verbs https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/5382c9d560f43ce6666c30f8aba47eef79f25969/engineering-processes.md#the-changelog

Can we add a `Removals` Section too?

Reply:
👍 updated it, thanks for flagging